subsys/bluetooth/mesh/scene_srv.c

```diff
@@ -122,29 +122,22 @@ static uint16_t target_scene(const struct bt_mesh_scene_srv *srv, int64_t now)
 	return BT_MESH_SCENE_NONE;
 }
 
-static void scene_status_encode(struct bt_mesh_scene_srv *srv,
-				struct net_buf_simple *buf,
-				enum bt_mesh_scene_status status)
+static void scene_status_encode(struct bt_mesh_scene_srv *srv, struct net_buf_simple *buf,
```

Comment:
```suggestion
static void scene_state_encode(struct bt_mesh_scene_srv *srv, struct net_buf_simple *buf,
```

Reply:
Should we change this name? What are we encoding? If is is the status message maybe we should keep this name, but if we change this we should also change the "scene_status_send" message tool